### DISCUSSION TITLE
Cant Push unreal project on github desktop with git lfs enabled

### DISCUSSION THREAD

**Discussion by redhood777** [community/community#79212] [New to GitHub] [answered]
I made an unreal project. In the middle of development I wanted other collaborators so I tried making a repository. Since the unreal project is large in size I purchased Git LFS. I set up the attributes and Git Ignore files correctly but whenever I try to push I get a "Remote Disconnected. Check internet connection". My internet is working fine, My ISP hasnt put upload limitations, I can make/push other git repositories easily but for the larger projects this issue is happening. I tried Dividing my changes into chunks and commiting them one at a time but still the issue persists. Please help.

**Comment by queenofcorgis** [comment]
Hi there are you using SSH?

**Reply by redhood777** [reply, marked as answer]
Not using an SSH, but i managed to fix it. Using unreal's in-editor source control(revision control) and connecting it to github worked for me.
https://www.youtube.com/watch?v=n3x1fErlmYA&t=879s

**Comment by Skylerspence** [comment]
Contact corporate it is an issue with github not your project.

**Comment by Skylerspence** [comment]
There not going to just put it on the platform you first need to have it reviewed
